impeller/renderer/backend/metal/surface_mtl.mm

```diff
@@ -61,7 +61,8 @@
   color0_resolve_tex_desc.format = color_format;
   color0_resolve_tex_desc.size = color0_tex_desc.size;
   color0_resolve_tex_desc.usage =
-      static_cast<uint64_t>(TextureUsage::kRenderTarget);
+      static_cast<uint64_t>(TextureUsage::kRenderTarget) |
```

Comment:
Like the `color_format` var above, I think we should read this from `current_drawable.texture.usage` instead of making it match in two places. Then, a validation can be added here saying the texture could not be read from.

Reply:
I suppose I am not sure why the usage of the resolve texture matters. Shouldn't it be on the usage flags of the color0_tex_desc descriptor?

Reply:
Changed this to not read from the surface texture.